docs/extending/extending_draftail.md

```diff
@@ -57,6 +57,18 @@ def register_mark_feature(features):
     # on rich text fields that do not specify an explicit 'features' list
     features.default_features.append('mark')
 ```
+Second, in `base.py` of the `settings` folder, configure `WAGTAILADMIN_RICH_TEXT_EDITORS` to display the block type in the toolbar by adding `mark` to `OPTIONS.features`
```

Comment:
@laymonage thanks for the response! I will make the changes!

Reply:
> The other one is wagtail.tests.test_migrations.TestForMigrations. test__migrations, which ensures that running python manage.py makemigrations will not create spurious migrations. This is because we still need to either (a) update the migrations or (b) create new migrations that drop and recreate the constraints for the test models.

Exactly! Perhaps, given the new direction (a) might be the better option since we still want to retain `unique_together`.

Reply:
@laymonage can you provide me the steps to create the migration? Thanks.